### PR TITLE
Typingdna activelock

### PR DESCRIPTION
**Pull request by raulpopa**
### What does this PR do?

TypingDNA ActiveLock Integration (closed previous PR because of a rebase issue, and reopened this clean one)

### Additional Notes

Original PR: https://github.com/DataDog/integrations-extras/pull/1572

### CODE REVIEW

File: typingdna_activelock/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# CHANGELOG - TypingDNA ActiveLock
+
+
+## 1.0.0 / 2022-10-30
```
Comment (emarsha94) [inline typingdna_activelock/CHANGELOG.md:4]:
```suggestion
## 1.0.0 / 2022-12-22
```

**Review by emarsha94 — APPROVED** [review]
Approved! Thanks for re-submitting this. Appreciate it!

File: typingdna_activelock/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Troubleshooting
+
+Need help? Contact [Datadog][1] or [TypingDNA support][2].
+
+[1]: https://docs.datadoghq.com/help/
+[2]: https://www.typingdna.com/contact
+[3]: https://www.typingdna.com/activelock
+[4]: https://app.datadoghq.com/organization-settings/api-keys
+[5]: https://app.datadoghq.com/logs
```
Comment (yzhan289) [inline typingdna_activelock/README.md:53]: Not too sure about how these links would work if a customer is not in the US. @emarsha94 Is this fine?

Comment (emarsha94) [reply typingdna_activelock/README.md:53]: We have provided this for other integrations so this should be fine!

**Review by yzhan289 — APPROVED** [review]

**Review by kayayarai — APPROVED** [review]

File: typingdna_activelock/manifest.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      "Category::Log Collection",
+      "Category::Monitoring",
+      "Category::Security"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "TypingDNA ActiveLock": "assets/dashboards/TypingDNAActiveLock.json"
+    },
+    "logs": {}
```
Comment (bgoldberg122) [inline typingdna_activelock/manifest.json:32]:
```suggestion
    }
```

**Review by bgoldberg122 — APPROVED** [review]
Looks good!

**Pull request merged**